config/service_providers.yml

```diff
@@ -161,6 +173,31 @@ production:
     attribute_bundle:
       - email
 
+# Micro-purchase
+  'urn:gov:gsa:saml:2.0.profiles:sp:sso:micropurchase-staging':
+    acs_url: 'https://micropurchase-staging.18f.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://micropurchase-staging.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://micropurchase-staging.18f.gov/login'
```

Comment:
this is not an actual URL in MP, I was just copying the rails demo sp (which also does not have this path defined). What is this config val used for? @pkarman you know?